x-pack/solutions/security/plugins/security_solution/public/reports/components/ai_value/value_report_settings.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback } from 'react';
+
+import { EuiText, EuiLink, EuiIcon } from '@elastic/eui';
+import { useNavigation } from '@kbn/security-solution-navigation';
+import { css } from '@emotion/react';
+import * as i18n from './translations';
+
+interface Props {
+  minutesPerAlert: number;
+  analystHourlyRate: number;
+}
+
+const ValueReportSettingsComponent: React.FC<Props> = ({ minutesPerAlert, analystHourlyRate }) => {
+  const { navigateTo } = useNavigation();
+
+  const goToKibanaSettings = useCallback(
+    () => navigateTo({ appId: 'management', path: '/kibana/settings?query=defaultValueReport' }),
+    [navigateTo]
+  );
+  return (
+    <EuiText size="xs" className="valueReportSettings">
+      <h3>{i18n.COST_CALCULATIONS}</h3>
+      <p>
+        {i18n.COST_CALCULATION({ minutesPerAlert, analystHourlyRate })}{' '}
+        <EuiLink onClick={goToKibanaSettings}>
+          {i18n.CHANGE_RATE}
```

Comment:
This is showing the 'open in new page' icon even though it opens in the current tab:

https://github.com/user-attachments/assets/0006409d-20b9-4ee4-a203-c5fc1f21618f